lib/propresenter_communication.js

```diff
@@ -96,10 +106,14 @@ var StageDisplay = function(host, port, password, identifier, onContentChange) {
     };
   };
 
-  var preprocessors = {
-    'clock': function(data) { return parseTime(data); }
+  var contentPreprocessors = {
+    'clock': function(data) { return parseTime(data); },
   };
 
+  var dataPreprocessors = {
+    'slide' : function(data) { return parseColors(data); }
```

Comment:
Is this enough or will the preprocessing be needed for clock, countdown and elapsed too?